packages/cdk-static-website/src/website-alias-record.ts

```diff
@@ -16,6 +17,9 @@ export interface WebsiteAliasRecordProps {
 
   /** Target for the alias record */
   readonly target: IAliasRecordTarget;
+
+  /** We support IPv6 and add an AAAA record by default, but you can turn it off */
+  readonly disableIPv6?: boolean;
```

Comment:
Why didn't you implement it as an opt-in feature?

Reply:
Because IPv6 has been around for 8 years, and increasingly our services should "Just Work" with it. I expect that the normal case for most people would be to want IPv6, and that if someone were to disable it, that'd be because they have a specific operational requirement or other exception that temporarily prevents them from using IPv6.

TL;DR: It's 2020 and I feel like IPv6 support should be the default, not something you opt in to. That's a subjective opinion, but that's why I implemented it this way.

Reply:
Alright thx